decorator/decorator.go

```diff
@@ -29,16 +29,17 @@ func NewDecorator(fset *token.FileSet) *Decorator {
 }
 
 // NewDecoratorWithImports returns a new decorator with import management enabled.
-func NewDecoratorWithImports(fset *token.FileSet, path string, resolver resolver.DecoratorResolver) *Decorator {
+func NewDecoratorWithImports(fset *token.FileSet, path string, resolver resolver.DecoratorResolver, includeLocalPkg bool) *Decorator {
```

Comment:
It might be better to switch to a config struct here. Let me know and I can add it.